appveyor.yml

```diff
@@ -0,0 +1,33 @@
+version: 3.0.0.{build}
+
+branches:
+    except:
+        - 26compat
+        - gh-pages
+        - legacy
+        - legacy-1.1-dev
+
+# Do not build on tags (GitHub only)
+skip_tags: true
+
+clone_folder: c:\build
+
+os: Visual Studio 2015
```

Comment:
They have an image called "Visual Studio 2015" and "Previous Visual Studio 2015". They do not have fine grained images. Here are a list of their updates: https://www.appveyor.com/updates